docs-client/src/lib/transports/transport.ts

```diff
@@ -63,6 +65,9 @@ export default abstract class Transport {
     const applicationType = httpResponse.headers.get('content-type') || '';
     if (applicationType.indexOf('json') >= 0) {
       try {
+        if (this.ignorePretty(applicationType)) {
```

Comment:
We should add this logic for the request body not for the response body. 😓 
https://github.com/line/armeria/blob/3af883e675201e0c119360bb786ac98b69db7246/docs-client/src/containers/MethodPage/DebugPage.tsx#L190